Svc/CmdSequencer/test/ut/Tester.cpp

```diff
@@ -43,13 +43,12 @@ namespace Svc {
         this->mallocator,
         BUFFER_SIZE
     );
-    this->component.preamble();
```

Comment:
Another behavioral change.

Reply:
This was cleaning up an unnecessary private method call. The test case was unnecessarily calling preamble, since this class itself didn't have a preamble implementation.